visidata/experimental/guide.py

```diff
@@ -86,4 +91,15 @@ def iterload(self):
                 yield [i] + list(m.groups())
                 i += 1
 
-BaseSheet.addCommand('', 'open-guide', 'vd.push(GuideGuide("VisiData_Guide"))')
+@VisiData.api
+def get_guide(vd, guidename):
```

Comment:
If we have `addGuide`, shouldn't this be `getGuide`?

Reply:
That....makes sense....